src/test/java/com/wikia/webdriver/common/core/Helios.java

```diff
@@ -18,18 +19,16 @@
 import org.json.JSONException;
 import org.json.JSONObject;
 
-import java.io.IOException;
-import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.List;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.logging.LOG;
+import com.wikia.webdriver.common.properties.HeliosConfig;
 
 /**
  * Created by Ludwik on 2015-08-05.
  */
 public class Helios {
 
-  private Helios() {
-  }
+  private Helios() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)